services/myteam.service.js

```diff
@@ -10,9 +10,8 @@ const getSupervisorsListForMyTeam = async (user) => {
 		if (myTeams) {
 			const supervisorList = myTeams.team.map((item) => {
 				//Until implement user roles and status
-
 				if (item.user.role === 'user' || true) {
```

Comment:
This is redundant and bad practise. use a middleware. Already implemented with routes in notifications or auth